fido2/client.py

```diff
@@ -271,20 +272,47 @@ class WEBAUTHN_TYPE(six.text_type, Enum):
 _CTAP1_INFO = b"\xa2\x01\x81\x66\x55\x32\x46\x5f\x56\x32\x03\x50" + b"\0" * 16
 
 
+def running_windows_non_admin():
+    """
+    Determine if running on Windows with version >= 1903 
+    and not as an administrator, which would require the use of
+    the built in WebAuthN API in Windows.
+    """
+    import platform
```

Comment:
i would put this import at the top of the file since it will always be used when initializing a FIDO2 client